### DISCUSSION TITLE
Cloudflare Zero Trust - Failed to fetch user/group information from the identity provider

### DISCUSSION THREAD

**Discussion by macmillernz** [authelia/authelia#3818] [Q&A]
After following this document https://www.authelia.com/integration/openid-connect/cloudflare-zerotrust/ I am unable to successfully test the OIDC.

It seems that the claims are not being sent. Does anyone have any suggestions?

The error I get while testing:
Failed to fetch user/group information from the identity provider
{
  "name": "oidc",
  "idpType": "oidc"
}

**Comment by james-d-elliott** [comment]
Any logs on either side? Authelia will log requests to the token endpoint when there is an error. You can enable debug logging to see more information. I suspect there's an issue with Cloudflare communicating with Authelia as I was the one who setup the docs for CFZT.
